providers/openmetrics/server_interceptor.go

```diff
@@ -1,35 +1,21 @@
 package metrics
```

Comment:
Server metrics and interceptor are refactored in the same way as client metrics and interceptor. All comments apply, not repeating them.